media/transcoder/src/test/java/com/example/transcoder/CreateJobWithStaticOverlayTest.java

```diff
@@ -0,0 +1,151 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.transcoder;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertNotNull;
+
+import com.google.api.gax.paging.Page;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.BlobInfo;
+import com.google.cloud.storage.Bucket;
+import com.google.cloud.storage.BucketInfo;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageClass;
+import com.google.cloud.storage.StorageOptions;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.UUID;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class CreateJobWithStaticOverlayTest {
+
+  private static final String LOCATION = "us-central1";
+  private static final String BUCKET_NAME =
+      "java-samples-transcoder-test-" + UUID.randomUUID().toString().substring(0, 25);
+  private static final String TEST_FILE_NAME = "ChromeCast.mp4";
+  private static final String TEST_FILE_PATH =
+      "src/test/java/com/example/transcoder/testdata/" + TEST_FILE_NAME;
+  private static final String INPUT_URI = "gs://" + BUCKET_NAME + "/" + TEST_FILE_NAME;
+  private static final String TEST_OVERLAY_NAME = "overlay.jpg";
+  private static final String TEST_OVERLAY_PATH =
+      "src/test/java/com/example/transcoder/testdata/" + TEST_OVERLAY_NAME;
+  private static final String OVERLAY_URI = "gs://" + BUCKET_NAME + "/" + TEST_OVERLAY_NAME;
+  private static final String OUTPUT_URI_FOR_AD_HOC = "gs://" + BUCKET_NAME + "/test-output-adhoc/";
+  private static String PROJECT_ID;
+  private static String PROJECT_NUMBER;
+  private static String JOB_ID;
+  private static PrintStream originalOut;
+  private ByteArrayOutputStream bout;
+
+  private static String requireEnvVar(String varName) {
+    String varValue = System.getenv(varName);
+    assertNotNull(
+        String.format("Environment variable '%s' is required to perform these tests.", varName));
+    return varValue;
+  }
+
+  private static void deleteBucket(String bucketName) {
+    Storage storage = StorageOptions.newBuilder().setProjectId(PROJECT_ID).build().getService();
+    Bucket bucket = storage.get(bucketName);
+    if (bucket != null) {
+      Page<Blob> blobs = bucket.list();
+
+      for (Blob blob : blobs.iterateAll()) {
+        System.out.println(blob.getName());
```

Comment:
is it necessary to print the blob names before deleting them?

Reply:
Removed